include/irr/asset/filters/CSummedAreaTableImageFilter.h

```diff
@@ -64,12 +60,8 @@ class CSummedAreaTableImageFilterBase
 				static inline size_t cachesOffset = {};
```

Comment:
unused variable

Reply:
oeo fo you use it for the image MIn/MAX ?

Reply:
I wanted it to be a cache for max (now I need also min)

Reply:
ok